packages/forma-36-react-components/src/components/Accordion/__snapshots__/Accordion.test.tsx.snap

```diff
@@ -73,8 +74,9 @@ exports[`renders the component with an additional class name 1`] = `
         <svg
           class="Icon Icon--small Icon--secondary Icon--trimmed AccordionHeader__icon"
           data-test-id="cf-ui-icon"
+          height="1em"
           viewBox="0 0 9 18"
-          xmlns="http://www.w3.org/2000/svg"
+          width="1em"
```

Comment:
Are these `width/height` changes expected? I see a lot of them in different svgs and snapshots

Reply:
They're from the way SVGR CLI generates components based on SVG files. I don't know if it would be preferable to keep the sizes locked with px?